library/Rules/Date.php

```diff
@@ -26,7 +26,7 @@ public function validate($input)
     {
         if ($input instanceof DateTime) {
```

Comment:
we got an issue here.

how could we compare if the given DateTime is compatible with the rule format (if provided)?